src/utils/MusicPlayerUtil.js

```diff
@@ -6,6 +6,28 @@ export const handlePlaylistChange = async (playlist) => {
 		await TrackPlayer.stop();
 		await TrackPlayer.reset();
 		await TrackPlayer.add(playlist);
+	} catch (error) {
+		Alert.alert("Something went wrong");
+	}
+};
+
+export const handleStartPlaylist = async (playlist) => {
+	try {
```

Comment:
use the existing method handle playlist change, and just add

Reply:
Reason i changed this....
Playlist starts null. So when a song gets pressed  playlist has to  change first, then index has to change before it plays. 

I figured it be easier to keep all action separated -----> press song -> handles playlist change (if needed) -> skips to index (if playlist song pressed) -> play song

if its from start playlist button -----> handles playlist change (if needed) -> starts playing playlist

Reply:
Let's talk about this during call